app/src/main/java/org/dhis2/uicomponents/map/geometry/mapper/featurecollection/MapDataElementToFeatureCollection.kt

```diff
@@ -0,0 +1,61 @@
+package org.dhis2.uicomponents.map.geometry.mapper.featurecollection
+
+import com.mapbox.geojson.Feature
+import com.mapbox.geojson.FeatureCollection
+import org.dhis2.data.dhislogic.CoordinateDataElementInfo
+import org.dhis2.uicomponents.map.geometry.bound.BoundsGeometry
+import org.dhis2.uicomponents.map.geometry.mapper.MapGeometryToFeature
+
+class MapDataElementToFeatureCollection(
+    private val mapGeometryToFeature: MapGeometryToFeature,
+    private val bounds: BoundsGeometry
```

Comment:
you don't need to pass the boundsGeomtry down in new classes. You have the class GetBoundingBox.

Check MapRelationshipToFeatureCollection :+1:

Reply:
I will change this in the PR with the attributes as some class names changed

Reply:
I'm a bit confuse hehe, better to go in order and merge this one? :sweat_smile:

Reply:
Yes, what I mean is that to display the attributes I changed for example this class MapDataElementToFeatureCollection to MapFieldToFeatureCollection so that it can be shared.
The attribute PR depends on this one, so it is safer to make these changes in that PR.